### ISSUE TITLE
Site Search

### ISSUE THREAD

**Issue by lakesta** [lakesta/oxiem-p1#3]
http://project.oxiem.com/public/index.php?path_info=projects/polyone27659-polyone-website-redesignreb-1/tasks/166 

Open - needs to be changed to another account; provided you code. Check sample for styling.

Is there a way you can point it at the dev site so that I can test that?
Not that I know of.
Also, does it give you any options to change text styling?
Nope.

http://www.cwcdefense.com/search.html?words=boards

**Comment by lakesta** [comment]
Updated, let me know if you want colors to change on different things - or fonts or sizes  We have a style sheet for search now so we can update accordingly.

**Comment by crystalolig** [comment]
sweet. I forgot about POD search, will have to enable.

**Comment by crystalolig** [comment]
Jake, it's still showing ads, can you check to see if this is the code used? It could have changed once we paid for the Google Search... I guess.

<script>
  (function() {
    var cx = '006644478918457306408:ozetlqtcczw';
    var gcse = document.createElement('script');
    gcse.type = 'text/javascript';
    gcse.async = true;
    gcse.src = (document.location.protocol == 'https:' ? 'https:' : 'http:') +
        '//cse.google.com/cse.js?cx=' + cx;
    var s = document.getElementsByTagName('script')[0];
    s.parentNode.insertBefore(gcse, s);
  })();
</script>
<gcse:searchresults-only></gcse:searchresults-only>

**Comment by lakesta** [comment]
Yeah, that is what is being used.  I am not seeing any ads.  I copied this page/code from polyone and just replaced the key.

**Comment by crystalolig** [comment]
sweet

On Tue, Oct 20, 2015 at 9:14 PM, Jake <notifications@github.com> wrote:

> Yeah, that is what is being used. I am not seeing any ads. I copied this
> page/code from polyone and just replaced the key.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lakesta/oxiem-p1/issues/3#issuecomment-149750676>.
>



-- 
CRYSTAL OLIG  interactive director
crystal@goupward.com • t  614.787.6676
cleveland • columbus • dayton • springfield

**Comment by crystalolig** [comment]
hi there
tested tonight and got "no results found" for anything. can you replicate?

**Comment by crystalolig** [comment]
Yikes look like I did see this before EEEEE!!!
